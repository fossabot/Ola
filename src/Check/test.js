import React from 'react'
import { Check } from './'
import renderer from 'react-test-renderer'

it('Default check is checkbox', () => {
  const tree = renderer
    .create(<Check>Accept terms</Check>)
    .toJSON()

  expect(tree).toMatchSnapshot()
})

it('Check type checkbox', () => {
  const tree = renderer
    .create(<Check type="checkbox">Accept terms</Check>)
    .toJSON()

  expect(tree).toMatchSnapshot()
})

it('Check type radio', () => {
  const tree = renderer
    .create(<Check type="radio">Accept terms</Check>)
    .toJSON()

  expect(tree).toMatchSnapshot()
})
