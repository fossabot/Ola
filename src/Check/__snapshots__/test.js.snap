// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check type checkbox 1`] = `
<label
  className="ola_check"
>
  <input
    className="ola_check-input"
    type="checkbox"
  />
  <div
    className="ola_check-label"
  >
    <div
      className="ola_check-label-content"
    >
      Accept terms
    </div>
  </div>
</label>
`;

exports[`Check type radio 1`] = `
<label
  className="ola_check"
>
  <input
    className="ola_check-input"
    type="radio"
  />
  <div
    className="ola_check-label"
  >
    <div
      className="ola_check-label-content"
    >
      Accept terms
    </div>
  </div>
</label>
`;

exports[`Default check is checkbox 1`] = `
<label
  className="ola_check"
>
  <input
    className="ola_check-input"
    type="checkbox"
  />
  <div
    className="ola_check-label"
  >
    <div
      className="ola_check-label-content"
    >
      Accept terms
    </div>
  </div>
</label>
`;
